scripts/test-extensions-unit.bat

```diff
@@ -57,85 +57,85 @@ if "%ADS_TEST_GREP%" == "" (
 
 set ALL_PLATFORMS_API_TESTS_EXTRA_ARGS=--disable-telemetry --crash-reporter-directory=%VSCODECRASHDIR% --no-cached-data --disable-updates --user-data-dir=%VSCODEUSERDATADIR% --remote-debugging-port=9222 --extensions-dir=%VSCODEEXTENSIONSDIR%
 
```

Comment:
Please Ignore this file, was made changes for local testing and added accidentally. removing it with next iteration...

Forgot the command that runs only single extension, thats why had to uncomment other and run required extensions tests.

Reply:
FYI - easier way to do this using test-extensions-unit.js. See https://github.com/microsoft/azuredatastudio/blob/main/test/README.md#unit-tests